sklearn/decomposition/_nmf.py

```diff
@@ -1252,8 +1362,9 @@ class NMF(TransformerMixin, BaseEstimator):
     Fevotte, C., & Idier, J. (2011). Algorithms for nonnegative matrix
     factorization with the beta-divergence. Neural Computation, 23(9).
     """
+
     @_deprecate_positional_args
-    def __init__(self, n_components=None, *, init=None, solver='cd',
+    def __init__(self, n_components=None, init=None, solver='cd',
```

Comment:
Why did you remove the `*,` ?

Reply:
I messed up with the first merge with upstream I suppose... fixed